bin/build

```diff
@@ -106,26 +106,7 @@ popd
 
 if [ -z "$NO_OPERATOR" ]; then
   pushd "$OPERATOR_DIR"
-    echo "Building manageiq-operator manifest..."
-    echo "$(date +"%Y%m%d%H%M%S")-$(git rev-parse HEAD)" > ./BUILD
-    cat ./BUILD
-    echo
-
-    image_build_args="--image-build-args --pull"
-    image_builder=""
-
-    if [ "$ARCH" = "s390x" ]; then
-      # HACK: Using buildah as, under the covers, this uses --format docker,
-      # which is needed to build v2 manifest builds. Normally, we'd pass
-      # --format docker as above as another arg to image-build-args, however,
-      # there is a bug in operator-sdk v0.18.2 where it cannot accept multiple
-      # args in image-build-args.
-      #
-      # See also https://github.com/operator-framework/operator-sdk/blob/v0.18.2/cmd/operator-sdk/build/cmd.go#L75-L99
-      image_builder="--image-builder buildah"
-    fi
```

Comment:
This is needed and is what I meant by needing `buildah` (I thought it was --format buildah, but it's --image-builder buildah).

Note that the new operator-sdk might _not_ need this as it was a hack to begin with as per the comment.

Reply:
Ok, as discussed... this hack was because we _couldn't_ pass `--format docker` and `-image-builder buildah` was our way to hack that value in because operator-sdk was constructing the docker build command line for us.  Since the makefile calls docker build directly, we don't have that restriction anymore.

Reply:
https://github.com/ManageIQ/manageiq-pods/pull/862